gradle/libs.versions.toml

```
@@ -46,6 +46,8 @@ androidx-test-ext-junit = { module = "androidx.test.ext:junit-ktx", version.ref
 androidx-test-uiautomator = "androidx.test.uiautomator:uiautomator:2.3.0"
 
 androidx-compose-ui = { module = "androidx.compose.ui:ui", version.ref = "jetpack-compose" }
+androidx-compose-ui-preview = { module = "androidx.compose.ui:ui-tooling", version.ref = "jetpack-compose" }
+androidx-compose-ui-tooling = { module = "androidx.compose.ui:ui-tooling-preview", version.ref = "jetpack-compose" }
```

I guess it was meant to be the other way around?
```suggestion
androidx-compose-ui-tooling = { module = "androidx.compose.ui:ui-tooling", version.ref = "jetpack-compose" }
androidx-compose-ui-tooling-preview = { module = "androidx.compose.ui:ui-tooling-preview", version.ref = "jetpack-compose" }
```

---

Woops fixed in 3956fb79c73ddd20f5e700188ef17b35083bb2ab